schemas/development/openservice.schema.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://raw.githubusercontent.com/xarf/schema-discussion/master/schemas/development/openservice.schema.json",
+  "title": "XARF BOTNET",
+  "description": "A format to report open service events.",
+  "allOf": [
+    {
+      "$ref": "xarf_shared.schema.json#/XarfBase"
+    },
+    {
+      "type": "object",
+      "properties": {
+        "Report": {
+          "allOf": [
+            {
+              "$ref": "xarf_shared.schema.json#/IpAndUrlBasedReport"
+            },
+            {
+              "type": "object",
+              "properties": {
+                "ReportClass": {
+                  "type": "string",
+                  "enum": ["Content"]
+                },
+                "ReportType": {
+                  "type": "string",
+                  "enum": ["OpenService"]
+                },
+                "Service": {
+                  "type": "string",
+                  "description": "name of the service (e.g. redis)"
+                },
+                "Protocol": {
```

Comment:
Is this needed?

Reply:
That was requested by one of our customers. It's not required and in most cases doesn't help much, because most services only use one of the protocols. We can also remove it completely and let users write it to "Custom".